src/Components/Buttons/index.js

```diff
@@ -2,14 +2,11 @@ import React from "react";
 
 import styles from "./index.module.scss";
 
-const Button = ({name,func,h,w}) => {
-
-    const handleClick = () => {
-        return func();
-    };
+const Button = ({name, func}) => {
```

Comment:
Please add spaces between curly brackets, if the object or destructor contains only one line.